app/views/dashboard.html

```diff
@@ -82,21 +81,23 @@ <h3 translate="map.selected.average-pass-rate">Average Pass Rate</h3>
       <p class="number good">51</p>
     </div>
   </div>
-  <div map-flyout-school class="map-flyout" ng-if="mapView=='schools' && selectedSchool|exists"></div>
-  <!-- end map flyout -->
-
+  <div
+    class="map-flyout"
+    ng-show="hoveredSchool !== null"
+    ng-include="'views/schoolFlyout.html'">
```

Comment:
Just beware -- the path has to be quoted in the quotes, `="'blah'"`, because angular treats it as an expression